splash/qtutils.py

```diff
@@ -166,7 +166,7 @@ def qt2py(obj, max_depth=100):
 
     # print(obj, obj.__class__)
 
-    if isinstance(obj, QString):
+    if isinstance(obj, (QString,QByteArray)):
         return unicode(obj)
```

Comment:
I wasn't being precise when I said "null return values". I really meant "return undefined", which I agree is much more common than `return null`. I will try to make the behavior of this new API the same as the behaviors of existing APIs.

I just don't understand how it works in the existing APIs. [evaluateJavascript](http://qt-project.org/doc/qt-4.8/qwebframe.html#evaluateJavaScript) returns a QVariant, and according to [this](https://bugs.webkit.org/show_bug.cgi?id=105441), QVariant doesn't distinguish between `undefined` and `null`. I'll have to do some digging to figure out what's going on here.